SamplesDashboard/SamplesDashboard/ClientApp/src/components/layout/Layout.tsx

```diff
@@ -2,9 +2,9 @@ import * as React from 'react';
 import { Container } from 'reactstrap';
 import NavMenu from './NavMenu';
 
-export default (props: { children?: React.ReactNode }) => (
+export default (props: { children?: React.ReactNode, setThemeMethod: any }) => (
```

Comment:
Can we type the `setThemeMethod` parameter?
```suggestion
export default (props: { children?: React.ReactNode, setThemeMethod: (theme: string) => void }) => (
```